admin/controllers/pupil.js

```diff
@@ -252,6 +269,8 @@ const getPrintPupils = async (req, res, next) => {
 module.exports = {
   getAddPupil,
   postAddPupil,
+  getAddMultiple,
```

Comment:
Likewise - this could be clearer: `getAddMultiplePupils`?

Reply:
done